llama-index-integrations/embeddings/llama-index-embeddings-jinaai/llama_index/embeddings/jinaai/base.py

```diff
@@ -170,6 +216,11 @@ def __init__(
             self._encoding_queries in VALID_ENCODING
         ), f"Encoding Queries parameter {self._encoding_documents} not supported. Please choose one of {VALID_ENCODING}"
 
+        if self._task:
```

Comment:
actually you could just let the API return the error. So u do not need to mantain the logic here